app/src/store/app.ts

```diff
@@ -1,16 +1,32 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { configureStore, Middleware } from '@reduxjs/toolkit';
 import { useDispatch } from 'react-redux';
 import { voiceReducer } from './voice';
 
-export const store = configureStore({
-  reducer: {
-    voice: voiceReducer,
-  },
-});
+export const logActionType: Middleware = () => (next) => (action) => {
+  console.log(
+    action.type.match(/\/rejected$/g)
```

Comment:
We will still be able to differentiate since the actual `action.type` value is also logged. For example, the logs will look something like this:
```
voice/getAccessToken/pending
voice/getAccessToken/rejected SOME_ERROR_PAYLOAD
```